src/zc/table/table.py

```diff
@@ -238,7 +243,7 @@ def __getitem__(self, key):
 
     def __nonzero__(self):
```

Comment:
Python 3 needs this to be called `__bool__`.  Usually we do that by defining either and assigning the other name.

The fact that tests didn't catch this mistake leads me to think the test suite is incomplete?  Or maybe we have a `__len__` that works just as well, just less efficiently.

Reply:
I've added coverage to tox. We currenly have 75% test coverage, this __nonzero__ / __bool__ is not covered. :(